.github/workflows/nix.yaml

```
@@ -1,51 +1,88 @@
+      - id: createGithubAppToken
+        uses: actions/create-github-app-token@v3
+        with:
+          client-id: ${{ vars.OPERATOR_APP_CLIENT_ID }}
+          private-key: ${{ secrets.OPERATOR_PRIVATE_KEY }}
+          permission-contents: read
+      - uses: shikanime-labs/actions/checkout@v9
+        with:
+          github-token: ${{ steps.createGithubAppToken.outputs.token }}
+      - uses: shikanime-labs/actions/nix/setup@v9
+        with:
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)